### PR TITLE
Merge `master` into latest MR

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "the-events-calendar",
+  "version": "4.6.24.1",
```
Comment (barryhughes) [inline package.json]: This gives us two version properties, we only want one though right?

File: readme.txt [hunk 2]
```diff
@@ -215,6 +215,12 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 == Changelog ==
 
+= [4.6.24.1] 2018-10-08 =
```
Comment (barryhughes) [inline readme.txt]: This places the 4.6.24.1 changelog above the 4.6.25 changelog entries ... we probably want it in the other order.

File: package.json (hunk 1 above)
Comment (barryhughes) [reply package.json]: (and for M18.15, this is the wrong one)

**Comment by mitogh** [comment]
Would you mind take another look thanks @barryhughes

**Pull request merged**